pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -469,6 +470,8 @@ func (r *ReconcileHyperConverged) ensureHco(req *hcoRequest) error {
 		r.ensureKubeVirtPriorityClass,
 		r.ensureKubeVirtConfig,
 		r.ensureKubeVirtStorageConfig,
+		r.ensureKubeVirtStorageRole,
```

Comment:
In addition to the general comment about not doing it from reconcile - if there is no other choice, I would call these two methods from within `ensureCDI` and not as part of this loop. This is something that is not monitors by HC as other resources.

Reply:
Thanks for the review.
My original motivation to write the code this way is that ensureKubeVirtStorageConfig is currently implemented as its own reconcile function.
If the intention in rotating the code is to "don't test if the rolebinding exists in every reconcile iteration", I don't think that this is a good idea.
It would violate our rule to write reconcile such that we reach eventual consistency. This might negatively affect updates, as one example.
If it's just to make things tidy, I should probably combine it with ensureKubeVirtStorageConfig too.